lib/node_modules/@stdlib/repl/lib/completer.js

```diff
@@ -51,6 +51,7 @@ var debug = logger( 'repl:completer' );
 * @returns {Array} normalized completion list
 */
 function normalize( list ) {
+	var uniqueList = [];
```

Comment:
Why is this variable necessary? Why don't you simply avoid the sort of the object keys? Sort order of `objectKeys` is defined by insertion order in newer JS engines and TC39, so not sure why you need to maintain a separate list.